### PR TITLE
fix(Checkbox): Checkbox should be vertically aligned with content

### PR DESCRIPTION
**Pull request by bcalvery**
**Before:**
image

**After:**
image

### CODE REVIEW

**Comment by mnajdova** [comment]
Let's add one example to potentially catch regressions for this in future.

**Review by codepretty — APPROVED** [review]

File: packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts [hunk 1]
```diff
@@ -53,6 +55,10 @@ const checkboxStyles: ComponentSlotStylesInput<CheckboxProps & CheckboxState, Ch
     margin: v.margin,
     padding: v.padding,
 
+    [`&.${Checkbox.slotClassNames.indicator}`]: {
+      marginTop: pxToRem(2),
```
Comment (layershifter) [inline packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts]: image

image

This override is really strange. I expect that `margin` variable will be overridden instead:

#### `checkboxStyles.ts`
```diff
-    [`&.${Checkbox.slotClassNames.indicator}`]: {
-      marginTop: pxToRem(2),
-    },
```
#### `checkboxVariables.ts`
```diff
-  margin: `${pxToRem(20)}, 0`, // THERE IS A MISTAKE, COMMA!
+  margin: `pxToRem(2) 0 0 0`,
```

**Review by layershifter — CHANGES_REQUESTED** [review]
Should be the same thing applied to toggle?

File: packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts (hunk 1 above)
Comment (bcalvery) [reply packages/react/src/themes/teams/components/Checkbox/checkboxStyles.ts]: Thank you for catching that. I wonder why that mistake was not causing any errors.

**Pull request merged**